packages/snap-store-mobx/src/Search/Stores/SearchResultStore.ts

```diff
@@ -478,10 +478,14 @@ export class VariantSelection {
 		let availableVariants = variants.data;
```

Comment:
Any reason not to just set this to filter out all available variants from the start?

```
let availableVariants = variants.data.filter((variant) => variant.available);
```

Then the if statement is unnecessary and we can remove `&& variant.available` from the for loop filter?

Reply:
Seems a bit odd that we were not already doing this... 🤔